### PR TITLE
130 default config

### LINKED ISSUE #130: Create default config
Content
The back-end and front-end should have a config.default example. The config they use should be gitignored and should contain a custom name so that multle developers can test on the same network.

Feature
- add config.default
- gitignore current config
- update documentation to create own config
- use custom name from config

### PR DESCRIPTION
**Pull request by mrlomar**
## Contents
- Added default config to back-end
- Added default config to front-end
- Updated readme for front and back-end
- Updated `.gitignore` to ignore config.json (front-end) room_config.json (back-end)

## Issue
Closes #130

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,9 +6,11 @@ __pycache__/
 back-end/pkg
 back-end/bin
 back-end/src/*
+back-end/resources/room_config.json
```
Comment (IssaHanou) [inline .gitignore]: why not ignore all other jsons as well?

Comment (mrlomar) [reply .gitignore]: Now we ignore all JSON files in /production

Comment (mrlomar) [reply .gitignore]: renaming room_config to default would result in default_config.json.default
this does not seem more intuitive than room_config.json(.default), maybe just config.json(.default)

**Comment by mrlomar** [comment]
@GwennanS I moved the manuals a separate folder in resources

**Pull request merged**